packages/docs-site/src/components/presenters/hero-banner/hero-banner.scss

```diff
@@ -1,35 +1,39 @@
-@import "@royalnavy/css-framework/index.scss";
+@import "@royalnavy/css-framework";
 
 .hero-banner {
-  background-color: color(primary, 800);
+  background: color(primary, 800);
```

Comment:
Any reason why we're generalising the background, rather than explicitly setting `background-color`?

Reply:
No reason this was changed and changed back after realising both footer and HeroBanner were already picking the same background colour. Might have been an issue using a system colour picker maybe there was some window shadow overlaying when he person who wrote the ticket colour picked.

Reply:
Changed back to `background-color`.